tutorial-901/README.md

```diff
@@ -0,0 +1,31 @@
+# NodeJs Integration
+
+The nodejs platform initiative uses pipelines to build zip files of nodejs apps, then pipelines should upload them into s3, then tell Hosting API that the artifact is ready
```

Comment:
Our customers have no idea what Hosting API is. We should word this as just the node just platform

Reply:
Node.js everywhere, not nodejs